core/shared/src/main/scala/scalaz/zio/IO.scala

```diff
@@ -443,15 +443,15 @@ sealed abstract class IO[+E, +A] extends Serializable { self =>
   /**
    * Repeats this action with the specified schedule until the schedule
    * completes, or until the first failure. In the event of failure the progress
-   * to date, together with the error, will be passed to the specified handler.
+   * to date, together with the failure value, will be passed to the specified handler.
```

Comment:
With [the new terminology](https://github.com/scalaz/scalaz-zio/issues/330), "failure" represents the consequence in terms of computation that can fail in many ways. The value produced in the error channel is a "checked error". I would just make that distinction wherever it makes sense. I think it does in this case.